packages/cli/lib/index.ts

```diff
@@ -36,7 +36,9 @@ class NangoCommand extends Command {
                 }
             }
 
-            await upgradeAction(debug);
+            if (isCI) {
```

Comment:
Don’t we went the upgrade action if it is not CI?

Reply:
yep indeed 🤦🏻